tests/testthat/test-assign_arg_list.R

```diff
@@ -0,0 +1,6 @@
+test_that("First argument must be a list",{
+  expect_error(assign_arg_list(c(1,2,3)))
+})
+test_that("All arguments should be named",{
+  expect_error(assign_arg_list(list(1,2)))
+})
```

Comment:
Should also check if it "works" not just if it has errors. In other words, "do the assignments take place in the proper environment"?

Reply:
Fixed.